app/util/db.ts

```diff
@@ -6,6 +6,11 @@ import { attachPaginate } from 'knex-paginate';
 import env from './env';
 import logger from './log';
 
+/**
+ * Batch size -- to avoid overly large SQL statements.
+ */
+export const batchSize = env.nodeEnv === 'development' ? 100 : 2000;
```

Comment:
Ok good to see with Postgres that we could insert 2000 at a time. I assume you tested both sqlite and postgres.

Reply:
Yup, tested with both.